src/types/model/linkToMedia.ts

```diff
@@ -14,5 +14,6 @@ export interface CustomTypeModelLinkToMediaField {
 		placeholder?: string
 		select: typeof CustomTypeModelLinkSelectType.Media
 		allowText?: boolean
+		repeat?: boolean
```

Comment:
❓ Are we allowing link to media fields to be repeatable? I thought we weren't, but correct me if I'm wrong.